cmd/cluster/powervs/destroy.go

```diff
@@ -22,6 +22,14 @@ func NewDestroyCommand(opts *core.DestroyOptions) *cobra.Command {
 		Short:        "Destroys a HostedCluster and its resources on PowerVS",
 		SilenceUsage: true,
 	}
+
+	cmd.Flags().StringVar(&opts.PowerVSPlatform.ResourceGroup, "resource-group", opts.PowerVSPlatform.ResourceGroup, "IBM Cloud Resource group")
+	cmd.Flags().StringVar(&opts.InfraID, "infra-id", opts.InfraID, "Cluster ID with which to tag IBM Cloud resources")
+	cmd.Flags().StringVar(&opts.PowerVSPlatform.BaseDomain, "base-domain", opts.PowerVSPlatform.BaseDomain, "Cluster's base domain")
+	cmd.Flags().StringVar(&opts.PowerVSPlatform.Region, "region", opts.PowerVSPlatform.Region, "IBM Cloud region. Default is us-south")
```

Comment:
why do we need region and VPCRegion?
why do we have 3 different things region, VPCRegion and Zone which has the same default "us-south"?

Reply:
Those are 3 diff fields, only for us-south, all 3 fields sharing the same value.
For osaka, region - osa, zone - osa21 & vpc-region - jp-osa.
https://cluster-api-ibmcloud.sigs.k8s.io/reference/regions-zones-mapping.html

Reply:
I see, thanks.

You mention default is "us-south", but I can't see that default being set?

Reply:
My bad, added now, thanks!